components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java

```diff
@@ -32,6 +32,7 @@
 import org.wso2.carbon.identity.application.common.model.*;
```

Comment:
This is already fixed in : https://github.com/wso2/carbon-identity/blob/master/components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java